### ISSUE TITLE
CLI Config + global options

### ISSUE THREAD

**Issue by swalkinshaw** [roots/trellis-cli#70]
It would be nice to have a config system for trellis-cli itself. This means both a global config (likely in homedir) and local config overrides per project (in `.trellis/config.yml`). Right now there's probably only one global flag which is disabling virtualenv. However this feature would also allow us to build some more verbose logging output.

The ideal solution to global flags and config values would follow the standard precedence feature of CLIs:
1. explicit CLI flag
2. local config (if exists)
3. global config

**Comment by robrecord** [comment]
It would be nice to define the path to the trellis instance in this directory. Eg. If I am in the project root, the site is in /site and trellis is in /trellis, allow me to invoke trellis-cli without having to change directory.

**Comment by swalkinshaw** [comment]
have you tried this? It should work as you expect

**Comment by robrecord** [comment]
TLDR; Yes it does, you're right.

I did try, but something must have been wrong.
I tried with a fresh install and it worked. So I overhauled my trellis installation with fresh files, and of course, it worked.

Thanks!
